### PR TITLE
Enhancement/skale 999 add test

### PR DESCRIPTION
**Pull request by elvisgiv**

### CODE REVIEW

**Review by dmytrotkk — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -7,59 +7,98 @@
...
 
-#### Yarn
-
-```bash
-yarn add @skalenetwork/skale-api
+## Usage
+```js
+const skale = require('@skalenetwork/skale.js');
+
+console.log(skale);
```
Comment (dmytrotkk) [inline README.md:25]: consider adding contract call invocation example here

File: README.md [hunk 2]
```diff
@@ -7,59 +7,98 @@
...
+* `PORT`: port of test geth
+* `ETH_ACCOUNT`: test account address
+* `ETH_PRIVATE_KEY`: test account privatekey
+ 
+Then:
+```bash
+npm run test
+```
 
+**If you do not want to save them to a file, you can run tests like this:
```
Comment (dmytrotkk) [inline README.md]: fix markdown style

**Review by dmytrotkk — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (elvisgiv) [reply README.md:25]: i'm not agree, we have a lot of examples in `examples` folder

File: README.md (hunk 2 above)
Comment (elvisgiv) [reply README.md]: ok

File: README.md (hunk 1 above)
Comment (dmytrotkk) [reply README.md:25]: ok, fine

**Review by dmytrotkk — APPROVED** [review]

**Pull request merged**